builders/interact/ethereum-api/dev-env/hardhat.md

```diff
@@ -0,0 +1,247 @@
+
+5. A menu will appear which will allow you to create a new project or use a sample project. For this example, you can choose **Create an empty hardhat.config.js**
+
+![Hardhat Create Project](/images/builders/interact/ethereum-api/dev-environments/hardhat/hardhat-1.png)
+
+
+This will create a Hardhat config file (`hardhat.config.js`) in your project directory.
+
+Once you have your Hardhat project, you can also install the [Ethers plugin](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-ethers){target=_blank}. This provides a convenient way to use the [Ethers.js](/builders/build/eth-api/libraries/ethersjs/){target=_blank} library to interact with the network. To install it, run the following command:
+
+```sh
+npm install @nomicfoundation/hardhat-ethers ethers@6
+```
+
+## The Contract File {: #the-contract-file }
+
+With your empty project created, next you are going to create a `contracts` directory. You can do so by running the following command:
+
+```sh
+mkdir contracts && cd contracts
+```
+
+The smart contract that you'll deploy as an example will be called `Box`, it will let you store a value that can be retrieved later. In the `contracts` directory, you can create the `Box.sol` file:
+
+```sh
+touch Box.sol
+```
+
+Open the file and add the following contract to it:
+
+```solidity
+// contracts/Box.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.1;
+
+contract Box {
+    uint256 private value;
+
+    // Emitted when the stored value changes
+    event ValueChanged(uint256 newValue);
+
+    // Stores a new value in the contract
+    function store(uint256 newValue) public {
+        value = newValue;
+        emit ValueChanged(newValue);
+    }
+
+    // Reads the last stored value
+    function retrieve() public view returns (uint256) {
+        return value;
+    }
+}
+```
+
+## Hardhat Configuration File {: #hardhat-configuration-file }
+
+Before you can deploy the contract to your ContainerChain, you'll need to modify the Hardhat configuration file and create a secure file to store your private key in.
+
+You can modify the `hardhat.config.js` file to use either the Tanssi Dancebox Demo EVM ContainerChain or your own ContainerChain:
+
+
+```js
+// 1. Import the Ethers plugin required to interact with the contract
+require('@nomicfoundation/hardhat-ethers');
+
+// 2. Add your private key that is funded with tokens of your ContainerChain
+const privateKey = 'INSERT_PRIVATE_KEY';
+
+module.exports = {
+  // 3. Specify the Solidity version
+  solidity: '0.8.1',
+  networks: {
+    // 4. Add the network specification for your Tanssi EVM ContainerChain
+    dancebox: {
+      url: '{{ networks.dancebox.rpc_url }}',
+      chainId: {{ networks.dancebox.chain_id }}, // Fill in the EVM ChainID for your ContainerChain
+      accounts: [privateKey]
+    }
+  }
+};
```

Comment:
can you please run prettier here? i think there are some trailing commas missing 🤔